shell/platform/linux/fl_text_input_plugin.cc

```diff
@@ -479,24 +517,39 @@ FlTextInputPlugin* fl_text_input_plugin_new(FlBinaryMessenger* messenger,
       g_object_new(fl_text_input_plugin_get_type(), nullptr));
 
   g_autoptr(FlJsonMethodCodec) codec = fl_json_method_codec_new();
-  self->channel =
+  FlTextInputPluginPrivate* priv = static_cast<FlTextInputPluginPrivate*>(
+      fl_text_input_plugin_get_instance_private(self));
+  priv->channel =
       fl_method_channel_new(messenger, kChannelName, FL_METHOD_CODEC(codec));
-  fl_method_channel_set_method_call_handler(self->channel, method_call_cb, self,
+  fl_method_channel_set_method_call_handler(priv->channel, method_call_cb, self,
                                             nullptr);
-  self->view = view;
+  priv->view = view;
 
   return self;
 }
 
 gboolean fl_text_input_plugin_filter_keypress(FlTextInputPlugin* self,
                                               GdkEventKey* event) {
   g_return_val_if_fail(FL_IS_TEXT_INPUT_PLUGIN(self), FALSE);
+  if (FL_TEXT_INPUT_PLUGIN_GET_CLASS(self)->filter_keypress) {
+    return FL_TEXT_INPUT_PLUGIN_GET_CLASS(self)->filter_keypress(self, event);
+  }
+  return FALSE;
+}
+
+// Implements FlTextInputPlugin::filter_keypress.
+gboolean fl_text_input_plugin_filter_keypress_default(FlTextInputPlugin* self,
```

Comment:
Private methods should be above public ones.

Reply:
Done.